src/main/java/frc/robot/commands/IntakeSequence.java

```diff
@@ -41,12 +48,11 @@ public IntakeSequence(Intake intake, Elevator elevator, Conveyor conveyor, Shoot
         new WaitUntilCommand(elevator::atDesiredPosition),
         new WaitUntilCommand(intake::atDesiredIntakePosition),
         new InstantCommand(intake::startRollers),
-        Conveyor.receive(conveyor),
-        new InstantCommand(
-            () -> {
-              intake.stopRollers();
-            },
-            intake),
-        new InstantCommand(() -> intake.setDesiredIntakePosition(IntakePosition.STOWED)));
+        new ParallelCommandGroup(
+            Conveyor.receive(conveyor),
+            new WaitUntilCommand(() -> intake.beamBreakSensor.get())), // TODO test true or false
+        new InstantCommand(intake::stopRollers, intake),
+        new InstantCommand(() -> intake.setDesiredIntakePosition(IntakePosition.STOWED)),
```

Comment:
I don't think we want to require the beam break to be triggered. If it gets into the conveyor then we should also be done. How about 
```suggestion
        new ParallelDeadlineGroup(
            Conveyor.receive(conveyor),
            new SequentialCommandGroup(
                new WaitUntilCommand(() -> intake.beamBreakSensor.get())), // TODO test true or false
                new InstantCommand(intake::stopRollers, intake),
                new InstantCommand(() -> intake.setDesiredIntakePosition(IntakePosition.STOWED)),
            )),
        new InstantCommand(intake::stopRollers, intake),
        new InstantCommand(() -> intake.setDesiredIntakePosition(IntakePosition.STOWED)),
```

Reply:
What Basel has written looks correct

Reply:
If I'm correct about where it is, the sensor will be triggered before `Conveyor.receive(conveyor)` is done, so I believe the intake will stop and go to stowed position prematurely?

Reply:
Intake can go in before conveyor is fully done